src/components/layouts/AboutLayout.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Head from "next/head";
+import Header from "../general/Header";
+import Footer from "../general/Footer";
+import { Container } from "@material-ui/core";
+import { makeStyles, ThemeProvider } from "@material-ui/core/styles";
+import aboutTheme from "../../themes/aboutTheme";
+
+const useStyles = makeStyles({
+  main: {
+    padding: 0,
+    marginTop: -16
+  }
+});
+
+export default function AboutLayout({ children }) {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Head>
+        <link
```

Comment:
We're no longer using Font Awesome (I think), so we can remove this.

Reply:
removed